servers/rendering/renderer_scene_cull.h

```diff
@@ -506,7 +506,11 @@ class RendererSceneCull : public RenderingMethod {
 				case Dependency::DEPENDENCY_CHANGED_DECAL:
 				case Dependency::DEPENDENCY_CHANGED_LIGHT:
 				case Dependency::DEPENDENCY_CHANGED_REFLECTION_PROBE: {
-					singleton->_instance_queue_update(instance, true, true);
+					//requires repairing
+					if (instance->indexer_id.is_valid()) {
+						singleton->_unpair_instance(instance);
+						singleton->_instance_queue_update(instance, true, true);
+					}
```

Comment:
Instead of changing the behaviour for mesh, particles, multimesh, decals and lights. You can just move the `case Dependency::DEPENDENCY_CHANGED_REFLECTION_PROBE:` below so that it is with `DEPENDENCY_CHANGED_LIGHT_SOFT_SHADOW_AND_PROJECTOR` and `DEPENDENCY_CHANGED_CULL_MASK`